sdk-workflows/pom.xml

```diff
@@ -0,0 +1,152 @@
+<project
+  xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.dapr</groupId>
+    <artifactId>dapr-sdk-parent</artifactId>
+    <version>1.9.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>dapr-sdk-workflows</artifactId>
+  <packaging>jar</packaging>
+  <version>1.9.0-SNAPSHOT</version>
```

Comment:
For other projects, we've been using a different version number for the workflow SDK to help indicate that it's not yet production quality. Should we do something similar for `dapr-sdk-workflows`? For example, like having the version be `0.1.0-SNAPSHOT`?

Reply:
Great question for maintainers more familiar with this codebase.  I am in favor of the approach. It is more defensible to have incremental merges that way as well.